reference/OpenAPI.json

```diff
@@ -71619,7 +71634,7 @@
         "tags": [
           "Customers"
         ],
-        "summary": "List Customer Activities",
+        "summary": "List Customer Activities [Deprecated]",
```

Comment:
To jeszcze trzeba dodać do `title` w `CUSTOMERS-List-Customer-Activities.md`, bo to chyba i tak tamtejsze `title` nadpisuje `summary`.